### ISSUE TITLE
Seeding User following chains?

### ISSUE THREAD

**Issue by Matthew-Nwen** [BlindGhosty/LSWA-Project#2]
Populate script works, but makes a bunch of unconnected users. How should we connect them? A good stretch goal would be to use a Fisher-Yates Algo to shuffle numbers then connect; otherwise we could start by just linking to the previous user.

**Comment by kpwill** [comment]
Yes! I believe Malini is working on implementing that right now, not 100% positive on the logic she is using to link them

**Comment by kpwill** [comment]
If we used Fisher-Yates to shuffle them, what would the next step be?
A follow-up question is: how many users should each user follow?

**Comment by Matthew-Nwen** [comment]
After shuffling, then these following chains should be saved. That's a good question; for now let's keep it at one.

**Comment by kpwill** [comment]
Okay gotcha.  In the short term that will work but we'll eventually have to change it because logics 2 and 5 rely on users following multiple people
